CHANGELOG.md

```diff
@@ -14,6 +14,53 @@ Changes by Version
 
 ### UI Changes
 
+1.17.0 (2020-02-22)
+------------------
+
+### Backend Changes
+
+#### Breaking Changes
+
+#### New Features
+
+* [tracegen] Add service name as a command line option ([#2080](https://github.com/jaegertracing/jaeger/pull/2080), [@kevinearls](https://github.com/kevinearls))
+* Allow the Configuration of Additional Headers on the Jaeger Query HTTP API ([#2056](https://github.com/jaegertracing/jaeger/pull/2056), [@joe-elliott](https://github.com/joe-elliott))
+* Warn about time adjustment in tags ([#2052](https://github.com/jaegertracing/jaeger/pull/2052), [@bobrik](https://github.com/bobrik))
+* Add CLI flags for Kafka batching params ([#2047](https://github.com/jaegertracing/jaeger/pull/2047), [@apm-opentt](https://github.com/apm-opentt))
+* Add common TLS configuration ([#1838](https://github.com/jaegertracing/jaeger/pull/1838), [@backjo](https://github.com/backjo))
```

Comment:
this is not new feature. It was refactoring PR

Reply:
Okay, I've moved it to Minor improvements.